source/_components/switch.xiaomi_plug.markdown

```diff
@@ -0,0 +1,106 @@
+---
+layout: page
+title: "Xiaomi Smart WiFi Socket and Smart Power Strip"
+description: "Instructions how to integrate your Xiaomi Smart WiFi Socket aka Plug or Xiaomi Smart Power Strip within Home Assistant."
+date: 2017-08-26 10:18
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: xiaomi.png
+ha_category: Switch
+ha_version: 0.53
+ha_iot_class: "Local Polling"
+---
+
+The `xiaomi_plug` switch platform allows you to control the state of your Xiaomi Smart WiFi Socket aka Plug or Xiaomi Smart Power Strip.
+
+Current supported features are `on`, `off`. If the device provides the current load it will be reported.
+
+## {% linkable_title Getting started %}
```

Comment:
I moved the instructions from the vacuum platform to the component. 

https://home-assistant.io/components/xiaomi/#retrieving-the-access-token

We don't want to maintain the same stuff at multiple locations.

Reply:
Ack.